### PR TITLE
Added route of profile, locations and users page on settings page (#238nhcq)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/157412043-d96eb414-ba91-4744-bd05-a35c8c11c869.mp4

File: src/views/Settings.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
     <ion-content>
       <main>
-        <ion-card>
+        <ion-card @click="() => router.push('/profile')">
```
Comment (dt2patel) [inline src/views/Settings.vue]: routing should be on click of details button on each card.

**Review by dt2patel — APPROVED** [review]

**Review by dt2patel — CHANGES_REQUESTED** [review]
Meant to request changes. Click action should be on details button. not card

File: src/views/Settings.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Settings.vue]: Ok sir, I have removed route from card and added to detail button

**Pull request merged**